### ISSUE TITLE
Chars ~ and @ look weird when used together

### ISSUE THREAD

**Issue by plagov** [JetBrains/JetBrainsMono#415]
First of all, thanks a lot for the great font! I started using it the same day JetBrains has announced it.

The issue I have encountered.

When using chars `~` followed by `@`, then they look a bit weird. For example, the string: `~@clean-up`:
![image](https://user-images.githubusercontent.com/16601852/108690347-eeb29f00-7502-11eb-9f98-8734a2d910b3.png)

**Comment by CodeCrazy-ywt** [comment]
I suggest these two characters remain how they originally look but be connected together.

**Comment by exorcist365** [comment]
This is a ligature and not a bug.

**Comment by Xerrion** [comment]
> This is a ligature and not a bug.

He didn't call it a bug.

**Comment by CodeCrazy-ywt** [comment]
Yes, it's a ligature. But this ligature is too different from original characters and needs to be modified.

**Comment by philippnurullin** [comment]
@plagov Thanks for the feedback! I'll see how this can be improved.

**Comment by CodeCrazy-ywt** [comment]
@philippnurullin I came up with an idea, just like this:
![无标题](https://user-images.githubusercontent.com/62426149/124287991-3d440a00-db83-11eb-8eaf-dfd5260055db.png)
Would it be better?

**Comment by philippnurullin** [comment]
@CodeCrazy-ywt Interesting suggestion! 
Need to test it in the small size. I'm worrying that the `˜` won't be recognizieble. But maybe it's ok like this.

**Comment by Xerrion** [comment]
> @CodeCrazy-ywt Interesting suggestion!
> Need to test it in the small size. I'm worrying that the `˜` won't be recognizieble. But maybe it's ok like this.

I think this is way more recognisable than the current one, where the `@` symbol just looks like a weird a trailing of. Also why would `~` be superscripted here.

**Comment by philippnurullin** [comment]
Changed the construction of `~@` in the latest commit. Please test.
You can grab files from here https://github.com/JetBrains/JetBrainsMono/tree/master/fonts

**Comment by CodeCrazy-ywt** [comment]
Looks much better than before. Thanks.
